src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -333,17 +278,14 @@ public int compare(final ProTerritory t1, final ProTerritory t2) {
     });
 
     // Log prioritized territories
-    for (final ProTerritory patd : prioritizedTerritories) {
+    for (final ProTerritory patd : attackOptions) {
```

Comment:
I usually go the toString() route which you could debate whether that is better or worse than pushing it into the logger. The issue here is that the ProTerritory objects in the attackOptions have lots of fields and I don't want to always log the same ones. Eventually I'd like to refactor ProTerritory into better chunks but until then I think I'll put this off.